core/src/test/scala/org/apache/spark/SparkConfSuite.scala

```diff
@@ -371,6 +371,23 @@ class SparkConfSuite extends SparkFunSuite with LocalSparkContext with ResetSyst
       assert(thrown.getMessage.contains(key))
     }
   }
+
+  test("SPARK-24566") {
```

Comment:
This test is useless. It tests copy-pasted codes and if someone changes the original codes, this will still pass. I prefer to not add this one.